docs/source/configuration/telemetry/instrumentation/spans.mdx

```diff
@@ -157,14 +157,20 @@ The `mode` option will be defaulted to `spec_compliant` in a future release, and
 By default spans are marked in error only if the http status code is different than 200. If you want to mark a span in error for other reason you can override the `otel.status_code` attribute which is responsible to mark a span in error or not.
 If it's in error then `otel.status_code` = `error`, if not it will be `ok`.
 
-Here is an example if you want to mark the `router` and `supergraph` span in error if you have a graphql error in the payload.
+## Span name
+
```

Comment:
The section below this says how to change the names of the spans, but we don't describe what the default span name is (at least right here in this text).  Could we add some text here saying something like:

```
By default, we will use a span naming convention that aligns with the current [blah].

```

Thoughts?

Reply:
Added ! Thanks !